exonum-java-binding/core/src/main/java/com/exonum/binding/core/storage/indices/ProofMapIndexProxy.java

```diff
@@ -285,15 +285,15 @@ private UncheckedMapProof getMultiKeyProof(Collection<? extends K> keys) {
   private native UncheckedMapProof nativeGetMultiProof(long nativeHandle, byte[] keys);
 
   /**
-   * Returns the root hash of the underlying Merkle-Patricia tree.
+   * Returns the index hash of this map.
```

Comment:
I'd clarify what an index hash is (it represents the complete state of this map. Any modifications to the stored entries affect the index hash).